AirCasting/SearchAndFollow/SearchLocation/PMSensorType.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Lunar on 12/04/2022.
+//
+
+import Foundation
+
+enum PMSensorType: Codable, CaseIterable, Identifiable {
```

Comment:
This is complicating things. Problem with this is that with each added unit you'd have to create a new enum type, and than adjust all the classes. Here we are combining a unit type with a sensor type, why not go with the good 'ol struct or even a tuple?

```Swift
struct Cośtam {
  let sensorType: SensorType // AB3and2, OpenAQ etc.
  let unitType: UnitType // PM2, Ozone etc.
}
```